_posts/2023-02-16-coprs-migration-to-github.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Copr's migration to GitHub
+author: Jiří Kyjovský
+layout: post
+---
+
+For a very long time, Copr mainliners have been considering migrating from
+Pagure to GitHub. The problem has been the constant problems with Pagure
+and even though GitHub is not opensource, it just works unlike Pagure. Not
```

Comment:
> it just works unlike Pagure

I'd with less harsh wording so people don't get mad at us :D


Maybe just "and even though GitHub is not opensource, it just works."